### PR TITLE
Add WooCommerce Services Banner to Extensions Pages - Take 2

### PR DESCRIPTION
**Pull request by v18**
This PR implements @olaolusoga's designs from p7bbVw-1tf-p2 (under the 'Where else?' section). cc @jeffstieler @nabsul

Here's what it looks like on the featured page:

screen shot 2017-05-30 at 11 55 42 pm

The content (title, description, image) on the featured page can still be changed using the featured json file, but has the defaults shown above.

On the shipping page:

screen shot 2017-05-30 at 11 55 52 pm

# Testing instructions
1. In /includes/admin/class-wc-admin-addons.php make sure to use the test json files (I hosted the files locally):
On line 27 use this [featured json file](http://cld.wthms.co/um7Np)
The banner is hardcoded on the shipping pages, so no more changes are required
2. Make sure to delete your WordPress transients before testing, you can use a plugin like [Transient Cleaner](https://wordpress.org/plugins/artiss-transient-cleaner/)
3. Go to `wp-admin/admin.php?page=wc-addons&section=featured` and `wp-admin/admin.php?page=wc-addons&section=shipping_methods` to see the changes

# Still to do
- [ ] get final boxes image from @olaolusoga 
- [ ] better responsive designs?
- [x] handle if plugin has been installed but not activated (currently runs into an error when using the plugin install URL)
- [x] potentially use background installer, cc @jeffstieler

### CODE REVIEW

**Review by mikejolley — APPROVED** [review]
Looks good to me. I'll hold off merge if those final items will be ready today.

File: includes/admin/class-wc-admin-addons.php
Comment (v18) [inline includes/admin/class-wc-admin-addons.php]: If we are going to show this to store owners in Canada, then we should update the description copy. Right now, it says:

> WooCommerce Services makes shipping a breeze. Print a label, take advantage of discounted shipping rates, and send tracking information to your customer right as you process your order, all from the convenience of your WordPress dashboard.

It's not currently possible to print a label, or use discounted rates in Canada. Only possible to show live rates.

**Comment by claudiosanches** [comment]
Both missing items will come in a future Pull Request, so good to merge right now.
Thanks.

**Pull request merged**
